src/components/ListItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { fromMilleToHours } from '../lib/helper';
+import { shoppingLists } from '../lib/shoppingListsCollection';
+
+const ListItem = ({ listItem, listId, itemId }) => {
+  const checkItem = () => {
+    shoppingLists()
+      .doc(listId)
+      .update({
+        [itemId]: { ...listItem, lastPurchased: new Date().toUTCString() },
+      });
+  };
+
+  const isChecked = fromMilleToHours(listItem.lastPurchased) < 24;
```

Comment:
💯 👍

Reply:
Nice work overall. Your solution to this issue was superb👌 @hajarNasr and @saraheisa 👏👏